presto-spi/src/test/java/com/facebook/presto/spi/block/TestingBlockEncodingSerde.java

```diff
@@ -26,6 +26,8 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Objects.requireNonNull;
 
+// This class is exactly the same as BlockEncodingManager.
```

Comment:
How can we know this remains truth tomorrow?

a test of some kind?

Reply:
I didn't add this. I just added a comment to explain the intention of the original author. I don't know how this can be guaranteed. I'm not sure how one can write such a test.

1. The other class is not visible in spi.
2. Even if it is, how does one assert that they are the same?

Reply:
If we mean that these classes are verbatim copies of each other, we could have a test that compares their sources (assuming tests are run from source tree). 
On higher level, we could check that they contain same set of encoding factories, etc.